pkgs/by-name/ku/kubectl-validate/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub, nix-update-script }:
+buildGoModule rec {
+  pname = "kubectl-validate";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "kubernetes-sigs";
+    repo = "kubectl-validate";
+    rev = "v${version}";
+    hash = "sha256-0lwN+3Cy7O9kX9dh8PjxmvdyKSZhBxcxy+b+ZtuDZaw=";
+  };
+
+  vendorHash = null;
```

Comment:
Why is this null?

Reply:
It means that `buildGoModule` shouldn't try to download the go dependencies as they are already vendored by the upstream project.

Reply:
Got it, thanks.